--- imgui/imgui_impl_sdl_gl3.cpp	2017-10-23 20:02:38.290386408 +0300
+++ imgui/imgui_impl_sdl_gl3_patched.cpp	2017-10-12 22:53:01.449799000 +0300
@@ -9,10 +9,15 @@
 #include "imgui.h"
 #include "imgui_impl_sdl_gl3.h"
 
-// SDL,GL3W
+// SDL,GL
 #include <SDL.h>
 #include <SDL_syswm.h>
-#include <GL/gl3w.h>    // This example is using gl3w to access OpenGL functions (because it is small). You may use glew/glad/glLoadGen/etc. whatever already works for you.
+#ifdef _WIN32
+#include "glad/glad.h"
+#else
+#include "GL/glcorearb.h"
+#endif
+
 
 // Data
 static double       g_Time = 0.0f;
