--- bullet/BulletDynamics/Dynamics/btRigidBody.h	2017-10-23 19:49:39.019065375 +0300
+++ bullet/BulletDynamics/Dynamics/btRigidBody_patched.h	2017-10-23 19:48:46.512188000 +0300
@@ -179,7 +179,7 @@
 
 	///btRigidBody constructor for backwards compatibility. 
 	///To specify friction (etc) during rigid body construction, please use the other constructor (using btRigidBodyConstructionInfo)
-	btRigidBody(	btScalar mass, btMotionState* motionState, btCollisionShape* collisionShape, const btVector3& localInertia=btVector3(0,0,0));
+	btRigidBody(	btScalar mass = 0, btMotionState* motionState = 0, btCollisionShape* collisionShape = 0, const btVector3& localInertia=btVector3(0,0,0));
 
 
 	virtual ~btRigidBody()
